mteb/benchmarks/get_benchmark.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import annotations
+
+import difflib
+
+import mteb.benchmarks.benchmarks as benchmark_module
+from mteb.benchmarks.benchmarks import Benchmark
+
+BENCHMARK_REGISTRY = {
+    inst.name: inst
+    for nam, inst in benchmark_module.__dict__.items()
```

Comment:
Again, this seems like a workaround for not having `catalogue`.

Reply:
I have considered it, but since most people are not familiar with this is fairly simple. Just grab all the Benchmarks from the module.

`catalogue` can be quite hard to debug as you have to make sure everything is imported in the right way, but if this was I package only you or I were maintaining I would probably go or it.

Reply:
Good points!